### ISSUE TITLE
CaIman

### ISSUE THREAD

**Issue by gviejo** [pynapple-org/pynapple#129]
See how to operates with Caiman

**Comment by EricThomson** [comment]
I'm currently finally coming around to this: using neuroconv to build a proper save functionality to replace Caiman's broken `save_NWB()` functionality.  Today I played with neuroconv and it works very well (with a minor terminological question from me: https://github.com/catalystneuro/roiextractors/issues/287). 

When I used neuroconv to save nwb and then load the result directly into pynapple. It works mostly fine. The nwb time series are `Neuropil`, `Deconvolved`, and `DfOverF`. Pynapple yields the following:

![nap_keys](https://github.com/pynapple-org/pynapple/assets/6643322/e3430f3a-07f1-477f-9fcc-190032e9ba7e)

It seems the dff field isn't getting parsed quite right 🤷‍♂️ I can share an ipynb or py file to illustrate this. Just highlighting this for now maybe we can look after Cosyne.
